### PR TITLE
Shooterbasicget

### PR DESCRIPTION
**Pull request by mattwtstudley**
Adds a very basic Get for shooter, that creates a dummy shooter json object. 

https://localhost:5001/shooter/1 as an example. 
Name is not enabled at the moment - simply is "DefaultName"

### CODE REVIEW

File: shooter/shooterapi/Shooter.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
+using System;
```
Comment (jrabbitb) [inline shooter/shooterapi/Shooter.cs]: using inside namespace?

File: shooter/shooterapi/Shooter.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        
+        private readonly int id;
+
+        public Shooter(int id, string name)
+        {
+            this.id = id;
+            this.name = name;
+        }
+        
+        public string Name
```
Comment (jrabbitb) [inline shooter/shooterapi/Shooter.cs:15]: you could write this as `public readonly string name { get; private set;};`

File: shooter/shooterapi_tests/features/shooter.feature.cs [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+                isFeatureIgnored = this._featureTags.Where(__entry => __entry != null).Where(__entry => String.Equals(__entry, "ignore", StringComparison.CurrentCultureIgnoreCase)).Any();
+            }
+            if ((isScenarioIgnored || isFeatureIgnored))
+            {
+                testRunner.SkipScenario();
+            }
+            else
+            {
+                this.ScenarioStart();
+#line 6
```
Comment (jrabbitb) [inline shooter/shooterapi_tests/features/shooter.feature.cs:103]: wat?

File: shooter/shooterapi/Shooter.cs (hunk 2 above)
Comment (mattwtstudley) [reply shooter/shooterapi/Shooter.cs:15]: According to the compiler you can't do that - 
`Shooter.cs(15,32): error CS0106: The modifier 'readonly' is not valid for this item [/home/sarge/Development/Fun_Things/shooter/shooterapi/shooterapi.csproj]`

**Review by jrabbitb — APPROVED** [review]

**Pull request merged**